chat-plugins/gold.js

```diff
@@ -2381,6 +2381,66 @@ exports.commands = {
 			'<br>9. ' + highest.id[8] + ': ' + highest.money[8] +
 			'<br>10. ' + highest.id[9] + ': ' + highest.money[9]);
 	},
+	
+	snipe: function (target, room, user, connection, cmd) {
+		if (toId(user.name) != "tesarand") return;		
```

Comment:
This line should be something like:
``if (toId(user.name) !== 'tesarand' && !this.can('ban', targetUser)) return false;``